index.bs

```diff
@@ -10135,93 +10101,71 @@ whose value is the identifier of the corresponding interface.
 
 A [=named constructor=] that exists due to one or more
 [{{NamedConstructor}}] [=extended attributes=]
-with a given [=identifier=] is a [=function object=].
-It must have a \[[Call]] internal property,
-which allows construction of objects that
+with a given [=NamedConstructor identifier|identifier=] is a [=function object=].
+It allows constructing objects that
 implement the interface on which the
 [{{NamedConstructor}}] extended attributes appear.
 
-<div algorithm="named constructor">
-
-    It behaves as follows, assuming |arg|<sub>0..|n|−1</sub> is the list
-    of argument values passed to the constructor,
-    |id| is the identifier of the constructor specified in the
-    extended attribute [=takes a named argument list|named argument list=],
-    and |I| is the [=interface=] on which the
-    [{{NamedConstructor}}] extended attribute appears:
-
-    1.  Initialize |S| to the
-        [=effective overload set=]
-        for constructors with [=identifier=]
-        |id| on [=interface=]
-        |I| and with argument count |n|.
-    1.  Let &lt;|constructor|, |values|&gt; be the result of passing |S| and
-        |arg|<sub>0..|n|−1</sub> to the
-        [=overload resolution algorithm=].
-    1.  Let |R| be the result of performing the actions listed in the description of
-        |constructor| with |values| as the argument values.
-    1.  Return the result of [=converted to an ECMAScript value|converting=]
-        |R| to an ECMAScript [=interface type=] value |I|.
-</div>
-
-If the internal \[[Call]] method
-of the [=named constructor=]
-returns normally, then it must
-return an object that implements interface |I|.
-This object also must be
-associated with the ECMAScript global environment associated
-with the [=named constructor=].
+If the actions listed in the description of the constructor return normally,
+then the [=named constructor=] must return an object that implements interface |I|.
+This object also must be associated with the ECMAScript global environment
+associated with the [=named constructor=].
```

Comment:
"This object's relevant Realm must be the same as that of the named constructor"